### PR TITLE
#2197 Docs: how to use CsvSinkTask

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2197 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/custom_connector.rst [hunk 1]
```diff
@@ -886,3 +886,133 @@ of CounterBuilder must be after initializing the connector/task.
...
+    boolean move(String sourcePath, String targetPath);
+
+    /** Stop using this storage. */
+    void close();
+  }
+
+.. note::
+  You don’t need to worry about the implementation of Storage. you can read the
+  :ohara-source:`source code <ohara-connector/src/main/scala/com/island/ohara/connector/ftp/FtpStorage.scala>`
+  to see how Ohara creates a FfpStorage.
```
Comment (vitojeng) [inline docs/src/custom_connector.rst]: My suggestion:
```
You can read the :ohara-source:`FtpStorage <file/path>` as an example to see how to implement your own Storage. 
```

File: docs/src/custom_connector.rst [hunk 2]
```diff
@@ -886,3 +886,133 @@ of CounterBuilder must be after initializing the connector/task.
...
+We all know that to make a strong and robust connector, you have to take care of
+a lot of details. In order to ensure that the connector works, we must also prepare
+a lot of tests. Connector developers will spend a lot of time on this.
+
+Therefore, we have encapsulated most of the logic in CsvSinkTask, which hides a lot of
+complex behaviors. Just provide a configuration object and a :ref:`Storage <connector-storage>`
+implementation to complete a sink connector. You can save time to enjoy other happy things.
+
+
+.. code-block:: java
```
Comment (vitojeng) [inline docs/src/custom_connector.rst]: Could we have a short description for this code block?

File: docs/src/custom_connector.rst [hunk 3]
```diff
@@ -886,3 +886,133 @@ of CounterBuilder must be after initializing the connector/task.
...
+Storage
+-------
+
+This interface defines some common methods for accessing the file system, such as checking
+for the existence of a file, creating a new file, or reading an exiting file, etc. Connector
+developers can follow this interface to implement different file systems, such as FTP, HDFS,
+SMB, Amazon S3, etc. So, just provide the implementation of Storage to CsvSinkTask and you
+can implement a :ref:`Sink Connector <connector-sink>` very quickly.
+
+.. code-block:: java
```
Comment (vitojeng) [inline docs/src/custom_connector.rst]: Could we have a short description for this code block?

File: docs/src/custom_connector.rst [hunk 4]
```diff
@@ -886,3 +886,133 @@ of CounterBuilder must be after initializing the connector/task.
...
+     * @return a Storage instance
+     */
+    public abstract Storage getStorage(TaskSetting setting);
+  }
+
+getConfig(TaskSetting setting)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CsvSinkTask must know some information in order to run normally, e.g. **topicsDir**
+the path to store the output files. In order to be the best friend of programmer, ohara
```
Comment (vitojeng) [inline docs/src/custom_connector.rst]: **ohara** --> **Ohara**

**Comment by vitojeng** [comment]
retry docs

**Comment by vitojeng** [comment]
retry

File: docs/src/custom_connector.rst (hunk 1 above)
Comment (eechih) [reply docs/src/custom_connector.rst]: Addressed this commit.

File: docs/src/custom_connector.rst (hunk 3 above)
Comment (eechih) [reply docs/src/custom_connector.rst]: Addressed this commit.

File: docs/src/custom_connector.rst (hunk 4 above)
Comment (eechih) [reply docs/src/custom_connector.rst]: Addressed this commit.

File: docs/src/custom_connector.rst (hunk 2 above)
Comment (eechih) [reply docs/src/custom_connector.rst]: Addressed this commit.

**Comment by eechih** [comment]
@vitojeng Addressed your comments, please review again, thanks.

@chia7712 I have attached a picture, please review again, thanks.

**Pull request merged**